### PR TITLE
Remove mentions of old versions of iOS from release notes.

### PR DESCRIPTION
**Pull request by peter-b**

### CODE REVIEW

File: docs/notes/platforms.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Currently, the supported versions of Xcode are:
 * Xcode 8.0 on MacOS X 10.11
 * Xcode 8.0 on MacOS 10.12
 
-It is also possible to set other versions of Xcode, to allow testing on a wider range of iOS simulators. For instance, on OS X 10.10 (Yosemite), you can add *Xcode 5.1* in the *Mobile Support* preferences, to let you test your stack on the *iOS Simulator 7.1*.
+It is also possible to set other versions of Xcode, to allow testing on a wider range of iOS simulators. For instance, on OS X 10.12 (Yosemite), you can add *Xcode 6.2* in the *Mobile Support* preferences, to let you test your stack on the *iOS Simulator 8.2*.
```
Comment (livecodepanos) [inline docs/notes/platforms.md]: This should be `MacOS 10.12 (Sierra)` (instead of `OS X 10.12 (Yosemite)`)

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 51501af

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 51501af9baa047a1834ef8ee367696163b8e7d97

**Comment by livecode-vulcan** [comment]
😎 test success 51501af9baa047a1834ef8ee367696163b8e7d97
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/321)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/326)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/321)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/321)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/327)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/321)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/321)

**Pull request merged**